### PR TITLE
feat: Add vc status check option for VP

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: pkg/restapi/verifier/operation/operations_test.go [hunk 1]
```diff
@@ -1653,7 +1694,8 @@ const (
 	prCardVC = `{
 	  "@context": [
 		"https://www.w3.org/2018/credentials/v1",
-		"https://w3id.org/citizenship/v1"
+		"https://w3id.org/citizenship/v1",
+        "https://w3id.org/vc-revocation-list-2020/v1"
```
Comment (troyronda) [inline pkg/restapi/verifier/operation/operations_test.go:1698]: indentation looks off?

File: test/bdd/testdata/university_certificate.json [hunk 2]
```diff
@@ -9,9 +9,5 @@
     "id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
     "name": "Example University"
   },
-  "issuanceDate": "2010-01-01T19:23:24Z",
-  "credentialStatus": {
-    "id": "https://example.gov/status/24",
-    "type": "CredentialStatusList2017"
```
Comment (troyronda) [inline test/bdd/testdata/university_certificate.json:15]: Why not the new credential status list?

Comment (fqutishat) [reply test/bdd/testdata/university_certificate.json:15]: it's was bug to put credentialStatus here so I remove it

File: test/bdd/features/thirdparty_verifier_e2e.feature [hunk 3]
```diff
@@ -20,14 +20,15 @@ Feature: Verifier verifiable credentials and presentations in third party endpoi
...
-      | certified_mill_test_report.json | https://verifier.interop.digitalbazaar.com/verifier/credentials | did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd | 28xXA4NyCQinSJpaZdSuNBM4kR2GqYb8NPqAtZoGCpcRYWBcDXtzVAzpZ9BAfgV334R2FC383fiHaWWWAacRaYGs | did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd#z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd | Ed25519Signature2018 | Ed25519 |
+      #| university_degree.json          | https://vc.transmute.world/v0.1.0/verify/credentials            | did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd | 28xXA4NyCQinSJpaZdSuNBM4kR2GqYb8NPqAtZoGCpcRYWBcDXtzVAzpZ9BAfgV334R2FC383fiHaWWWAacRaYGs | did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd#z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd | Ed25519Signature2018 | Ed25519 |
```
Comment (troyronda) [inline test/bdd/features/thirdparty_verifier_e2e.feature:23]: These changes are due to those endpoints not supporting revocation list 2020?

File: pkg/restapi/verifier/operation/operations_test.go (hunk 1 above)
Comment (fqutishat) [reply pkg/restapi/verifier/operation/operations_test.go:1698]: I'm not sure why it's showing like this. looks good in goland

File: test/bdd/features/thirdparty_verifier_e2e.feature (hunk 3 above)
Comment (fqutishat) [reply test/bdd/features/thirdparty_verifier_e2e.feature:23]: no it's was failing for couple of weeks in master

Comment (fqutishat) [reply test/bdd/features/thirdparty_verifier_e2e.feature:23]: @troyronda master build https://github.com/trustbloc/edge-service/runs/1681418827

File: test/bdd/testdata/university_certificate.json (hunk 2 above)
Comment (troyronda) [reply test/bdd/testdata/university_certificate.json:15]: ok

**Review by troyronda — APPROVED** [review]

**Pull request merged**